index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mulugeta-Portfolio</title>
+    <link rel="stylesheet" href="style/styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100&display=swap" rel="stylesheet">
+</head>
+<body>
+    <!-- define container that holds all the items of the page-->
+    <div class="conatianer">
+
+        <!--Header part of the page-->
+        <header>
+            <p class="logo"> My Logo</p>
```

Comment:
- [ ] Great job with your code 👍 . Please make sure you wrap your logo with an `a` tag instead of a `p` tag because your logo is meant to link somewhere 🙏 .